src/core/runtime.d

```diff
@@ -342,9 +346,44 @@ extern (C) void dmd_coverDestPath(string path);
  *
  * Params:
  *  flag = enable/disable coverage merge mode
+ * Note:
+ *  This is a dmd specific setting.
  */
 extern (C) void dmd_coverSetMerge(bool flag);
 
+/**
+ * Set the output file name for profile reports (-profile switch).
+ * An empty name will set the output to stdout.
+ *
+ * Params:
+ *  name = file name
+ * Note:
+ *  This is a dmd specific setting.
+ */
+extern (C) void trace_setlogfilename(string name);
```

Comment:
Let's not repeat the discussion https://github.com/D-Programming-Language/druntime/pull/1225#issuecomment-104062444.